onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -749,7 +749,7 @@ extern "C" {
 
 #define MLAS_SGEMM_THREAD_COMPLEXITY                (size_t(64) * size_t(1024))
 #define MLAS_DGEMM_THREAD_COMPLEXITY                (size_t(64) * size_t(1024))
```

Comment:
Does the above two lines need change as well?

Reply:
tried constexpr, even more warnings appeared